src/main/webapp/app/controllers/discoveryContextController.js

```diff
@@ -50,7 +50,7 @@ sage.controller('DiscoveryContextController', function ($controller, $scope, $ro
           $location.search("page", null);
         }
 
-        if (angular.isDefined($scope.discoveryContext.sortFields)) {
+        if (angular.isDefined($scope.discoveryContext.sortFields) && $scope.discoveryContext.sortFields.length > 0 ) {
```

Comment:
There is extra whitespace at the between the `0` and the `)` for both this line and line 69.

Reply:
Removed whitespace